src/main/java/io/codelex/parbaudesDarbs/exercise4/FileReaderAndReverseWriter.java

```diff
@@ -0,0 +1,36 @@
+package io.codelex.parbaudesDarbs.exercise4;
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class FileReaderAndReverseWriter {
```

Comment:
Ok, šis man liekas strādā par 99%, jo visu izdara, izņemot apgriež faila nosaukumu otrādi, bet nu vari atstāt arī šādi..